components/post_view/channel_intro_message/__snapshots__/channel_intro_message.test.tsx.snap

```diff
@@ -149,16 +138,30 @@ exports[`components/post_view/ChannelIntroMessages test DEFAULT Channel should m
       }
     />
   </p>
-  <button
-    aria-label="Create a board"
-    className="intro-links color--link channelIntroButton style--none"
-    onClick={[Function]}
-  >
-    <MemoizedFormattedMessage
-      defaultMessage="Create a board"
-      id="intro_messages.createBoard"
-    />
-  </button>
+  <Memo(BoardsButton)
```

Comment:
Here, the button will show because the following is passed the `BoardsButton` as a prop.

```ts
    boardComponent={
      Object {
        "id": "board",
        "pluginId": "board",
      }
    }
```